nebius/terraform/infra/main.tf

```
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    nebius = {
+      source  = "terraform-provider-nebius.storage.ai.nebius.cloud/nebius/nebius"
```

```suggestion
      source  = "terraform-provider.storage.eu-north1.nebius.cloud/nebius/nebius"
```

Gotta update the url here too